make distcheck broken

```
$ make distcheck
...
Making all in src
Making all in tk
Making all in widget
  CC    button.o
  CC    checkbox.o
  CC    cust.o
  CC    fader.o
  CC    image.o
  CC    imagearray.o
  CC    input.o
  CC    list.o
  CC    rect.o
  CC    tabwin.o
  CC    text.o
ar: `u' modifier ignored since `D' is the default (see `U')
  CC    ai.o
cc1: fatal error: ../src/attributes.h: No such file or directory
compilation terminated.
make[4]: *** [Makefile:925: ai.o] Error 1
make[3]: *** [Makefile:963: all-recursive] Error 1
make[2]: *** [Makefile:661: all] Error 2
make[1]: *** [Makefile:661: all-recursive] Error 1
make: *** [Makefile:879: distcheck] Error 1
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -38,10 +38,12 @@ if HAVE_NLS
 XMLFILES    = $(top_srcdir)/po/POTFILES_XML.in
 TRANSLATIONITS = $(top_srcdir)/po/its/translation.its
 CREDITSPOT  = $(top_builddir)/po/credits.pot
-CREDITSTEXT = $(top_builddir)/dat/AUTHORS $(top_builddir)/dat/intro
+CREDITSTEXT = $(top_srcdir)/dat/AUTHORS $(top_srcdir)/dat/intro
+NAEVSRCPOT  = $(top_builddir)/po/$(GETTEXT_PACKAGE).pot
+NAEVPOT     = $(top_builddir)/po/$(PACKAGE).pot
 XMLPOT      = $(top_builddir)/po/xml.pot
 EXTRA_DIST += $(XMLFILES) $(TRANSLATIONITS)
-CLEANFILES += $(CREDITSPOT) $(XMLPOT)
+CLEANFILES += $(CREDITSPOT) $(NAEVPOT) $(XMLPOT)
 endif # HAVE_NLS
 
 .PHONY: data pot docs utils help install-ndata soundtrack
@@ -52,6 +54,7 @@ data: pot $(VERSIONFILE) $(LICENSEFILE)
 
 $(LICENSEFILE): $(top_srcdir)/LICENSE
 	@echo "  CP    $<"
+	@mkdir -p $(top_builddir)/dat
 	cp $< $@
 
 if HAVE_NLS
@@ -74,10 +77,12 @@ endif # HAVE_ITSTOOL
 if HAVE_DATA
 pot: $(CREDITSPOT) $(XMLPOT) $(LANGUAGEFILE)
 	$(MAKE) -C po
-	$(MAKE) -C po $(PACKAGE).pot
-	$(top_srcdir)/utils/gettext.sh $(top_srcdir)
+	$(MAKE) -C po naev_src.pot
+	$(XGETTEXT) --from-code=UTF-8 -o $(NAEVPOT) $(NAEVSRCPOT) $(XMLPOT) $(CREDITSPOT)
+	$(top_srcdir)/utils/gettext.sh $(top_builddir)
 
 $(LANGUAGEFILE): $(top_srcdir)/po/LINGUAS
+	@mkdir -p $(top_builddir)/dat
 	tail -n +2 $< > $@
 else # HAVE_DATA
 pot:
@@ -95,6 +100,7 @@ $(NAEV):
 	$(LN_S) "src/$(NAEV)" "$(NAEV)"
 
 $(VERSIONFILE): src/config.h
+	@mkdir -p $(top_builddir)/dat
 	@echo $(VERSION) > $@
 
 if HAVE_UTILS

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,11 +1,8 @@
 ACLOCAL_AMFLAGS = --install -I m4
 
 SUBDIRS = lib src
-DIST_SUBDIRS = lib src
+DIST_SUBDIRS = lib src utils
 
-if HAVE_UTILS
-SUBDIRS += utils
-endif # HAVE_NLS
 if HAVE_NLS
 SUBDIRS += po
 DIST_SUBDIRS += po
@@ -22,9 +19,7 @@ dist_pkgdata_DATA = naev-confupdate.sh
 dist_man6_MANS = naev.6
 dist_doc_DATA = README TODO LICENSE
 
-if HAVE_UTILS
 MKSPR      = $(top_builddir)/utils/mkspr/mkspr$(EXEEXT)
-endif
 NAEV	     = $(top_builddir)/$(PACKAGE)$(EXEEXT)
 
 VERSIONFILE = $(top_builddir)/dat/VERSION
@@ -103,14 +98,10 @@ $(VERSIONFILE): src/config.h
 	@mkdir -p $(top_builddir)/dat
 	@echo $(VERSION) > $@
 
-if HAVE_UTILS
 utils: $(MKSPR)
 
 $(MKSPR):
 	$(MAKE) -C utils/mkspr
-else # HAVE_UTILS
-utils:
-endif # HAVE_UTILS
 
 install-data-local:
 	(cd $(top_srcdir)/dat; find . -type f -exec install -Dm 644 "{}" "$(DESTDIR)$(pkgdatadir)/dat/{}" \;)

--- a/configure.ac
+++ b/configure.ac
@@ -75,8 +75,7 @@ AC_PROG_OBJC
 AC_PROG_LN_S
 AC_PROG_RANLIB
 AM_PROG_AR
-# Should be AX_PROG_CC_FOR_BUILD -- except Steam builds occur in a paleozoic environment where that doesn't work.
-AX_CC_FOR_BUILD
+AX_PROG_CC_FOR_BUILD
 PKG_PROG_PKG_CONFIG
 
 AC_CHECK_PROG([CUT], [cut], [cut])

--- a/configure.ac
+++ b/configure.ac
@@ -92,14 +92,6 @@ AC_CHECK_PROG([ITSTOOL], [itstool], [itstool])
 #
 # Checks for build environment
 #
-AC_MSG_CHECKING([whether we are building utils])
-AS_IF([test -d "$srcdir/utils"], [
-  have_utils=yes
-], [
-  have_utils=no
-])
-AC_MSG_RESULT([$have_utils])
-
 AC_MSG_CHECKING([whether we have data])
 AS_IF([test -d "$srcdir/dat"], [
   have_data=yes
@@ -580,7 +572,6 @@ AM_CONDITIONAL([HAVE_WINDRES], [test -n "$WINDRES"])
 AM_CONDITIONAL([HAVE_MACOS], [test "x$have_macos" = "xyes"])
 AM_CONDITIONAL([HAVE_DOXYGEN], [test -n "$DOXYGEN"])
 AM_CONDITIONAL([HAVE_LDOC], [test -n "$LDOC"])
-AM_CONDITIONAL([HAVE_UTILS], [test "x$have_utils" = "xyes"])
 AM_CONDITIONAL([HAVE_DATA], [test "x$have_data" = "xyes"])
 AM_CONDITIONAL([HAVE_DOCS], [test "x$have_docs" = "xyes"])
 AM_CONDITIONAL([HAVE_MKSPR], [test "x$have_mkspr" = "xyes"])
@@ -604,11 +595,9 @@ AC_CONFIG_FILES([Makefile
       src/Makefile
       src/tk/Makefile
       src/tk/widget/Makefile
-      build/shave])
-AS_IF([test "x$have_utils" = "xyes"], [
-  AC_CONFIG_FILES([utils/Makefile
-     utils/mkspr/Makefile])
-])
+      build/shave
+      utils/Makefile
+      utils/mkspr/Makefile])
 AS_IF([test "x$have_docs" = "xyes"], [
   AC_CONFIG_FILES([docs/Makefile])
 ])
@@ -632,7 +621,6 @@ AC_MSG_NOTICE([lib:])
 AC_MSG_NOTICE([  csparse:    $used_csparse])
 AC_MSG_NOTICE([  lua:        $used_lua])
 echo
-AC_MSG_NOTICE([utils:        $have_utils])
 AC_MSG_NOTICE([  mkspr:      $have_mkspr])
 echo
 AC_MSG_NOTICE([debug mode:   $enable_debug])

--- a/configure.ac
+++ b/configure.ac
@@ -15,7 +15,8 @@ AC_CONFIG_AUX_DIR([build])
 AM_GNU_GETTEXT([external])
 # Currently steam packages 0.19.7 which is the limiting factor here
 AM_GNU_GETTEXT_VERSION([0.19.7])
-GETTEXT_PACKAGE=${PACKAGE_NAME}
+# The actual ${PACKAGE_NAME}.pot file is a combination of the below, a credits.pot file, an an xml.pot file.
+GETTEXT_PACKAGE=${PACKAGE_NAME}_src
 AC_SUBST(GETTEXT_PACKAGE)
 
 IT_PROG_INTLTOOL

--- a/m4/ax_prog_cc_for_build.m4
+++ b/m4/ax_prog_cc_for_build.m4
@@ -1,0 +1,125 @@
+# ===========================================================================
+#   https://www.gnu.org/software/autoconf-archive/ax_prog_cc_for_build.html
+# ===========================================================================
+#
+# SYNOPSIS
+#
+#   AX_PROG_CC_FOR_BUILD
+#
+# DESCRIPTION
+#
+#   This macro searches for a C compiler that generates native executables,
+#   that is a C compiler that surely is not a cross-compiler. This can be
+#   useful if you have to generate source code at compile-time like for
+#   example GCC does.
+#
+#   The macro sets the CC_FOR_BUILD and CPP_FOR_BUILD macros to anything
+#   needed to compile or link (CC_FOR_BUILD) and preprocess (CPP_FOR_BUILD).
+#   The value of these variables can be overridden by the user by specifying
+#   a compiler with an environment variable (like you do for standard CC).
+#
+#   It also sets BUILD_EXEEXT and BUILD_OBJEXT to the executable and object
+#   file extensions for the build platform, and GCC_FOR_BUILD to `yes' if
+#   the compiler we found is GCC. All these variables but GCC_FOR_BUILD are
+#   substituted in the Makefile.
+#
+# LICENSE
+#
+#   Copyright (c) 2008 Paolo Bonzini <bonzini@gnu.org>
+#
+#   Copying and distribution of this file, with or without modification, are
+#   permitted in any medium without royalty provided the copyright notice
+#   and this notice are preserved. This file is offered as-is, without any
+#   warranty.
+
+#serial 9
+
+AU_ALIAS([AC_PROG_CC_FOR_BUILD], [AX_PROG_CC_FOR_BUILD])
+AC_DEFUN([AX_PROG_CC_FOR_BUILD], [dnl
+AC_REQUIRE([AC_PROG_CC])dnl
+AC_REQUIRE([AC_PROG_CPP])dnl
+AC_REQUIRE([AC_EXEEXT])dnl
+AC_REQUIRE([AC_CANONICAL_HOST])dnl
+
+dnl Use the standard macros, but make them use other variable names
+dnl
+pushdef([ac_cv_prog_CPP], ac_cv_build_prog_CPP)dnl
+pushdef([ac_cv_prog_gcc], ac_cv_build_prog_gcc)dnl
+pushdef([ac_cv_prog_cc_works], ac_cv_build_prog_cc_works)dnl
+pushdef([ac_cv_prog_cc_cross], ac_cv_build_prog_cc_cross)dnl
+pushdef([ac_cv_prog_cc_g], ac_cv_build_prog_cc_g)dnl
+pushdef([ac_cv_exeext], ac_cv_build_exeext)dnl
+pushdef([ac_cv_objext], ac_cv_build_objext)dnl
+pushdef([ac_exeext], ac_build_exeext)dnl
+pushdef([ac_objext], ac_build_objext)dnl
+pushdef([CC], CC_FOR_BUILD)dnl
+pushdef([CPP], CPP_FOR_BUILD)dnl
+pushdef([CFLAGS], CFLAGS_FOR_BUILD)dnl
+pushdef([CPPFLAGS], CPPFLAGS_FOR_BUILD)dnl
+pushdef([LDFLAGS], LDFLAGS_FOR_BUILD)dnl
+pushdef([host], build)dnl
+pushdef([host_alias], build_alias)dnl
+pushdef([host_cpu], build_cpu)dnl
+pushdef([host_vendor], build_vendor)dnl
+pushdef([host_os], build_os)dnl
+pushdef([ac_cv_host], ac_cv_build)dnl
+pushdef([ac_cv_host_alias], ac_cv_build_alias)dnl
+pushdef([ac_cv_host_cpu], ac_cv_build_cpu)dnl
+pushdef([ac_cv_host_vendor], ac_cv_build_vendor)dnl
+pushdef([ac_cv_host_os], ac_cv_build_os)dnl
+pushdef([ac_cpp], ac_build_cpp)dnl
+pushdef([ac_compile], ac_build_compile)dnl
+pushdef([ac_link], ac_build_link)dnl
+
+save_cross_compiling=$cross_compiling
+save_ac_tool_prefix=$ac_tool_prefix
+cross_compiling=no
+ac_tool_prefix=
+
+AC_PROG_CC
+AC_PROG_CPP
+AC_EXEEXT
+
+ac_tool_prefix=$save_ac_tool_prefix
+cross_compiling=$save_cross_compiling
+
+dnl Restore the old definitions
+dnl
+popdef([ac_link])dnl
+popdef([ac_compile])dnl
+popdef([ac_cpp])dnl
+popdef([ac_cv_host_os])dnl
+popdef([ac_cv_host_vendor])dnl
+popdef([ac_cv_host_cpu])dnl
+popdef([ac_cv_host_alias])dnl
+popdef([ac_cv_host])dnl
+popdef([host_os])dnl
+popdef([host_vendor])dnl
+popdef([host_cpu])dnl
+popdef([host_alias])dnl
+popdef([host])dnl
+popdef([LDFLAGS])dnl
+popdef([CPPFLAGS])dnl
+popdef([CFLAGS])dnl
+popdef([CPP])dnl
+popdef([CC])dnl
+popdef([ac_objext])dnl
+popdef([ac_exeext])dnl
+popdef([ac_cv_objext])dnl
+popdef([ac_cv_exeext])dnl
+popdef([ac_cv_prog_cc_g])dnl
+popdef([ac_cv_prog_cc_cross])dnl
+popdef([ac_cv_prog_cc_works])dnl
+popdef([ac_cv_prog_gcc])dnl
+popdef([ac_cv_prog_CPP])dnl
+
+dnl Finally, set Makefile variables
+dnl
+BUILD_EXEEXT=$ac_build_exeext
+BUILD_OBJEXT=$ac_build_objext
+AC_SUBST(BUILD_EXEEXT)dnl
+AC_SUBST(BUILD_OBJEXT)dnl
+AC_SUBST([CFLAGS_FOR_BUILD])dnl
+AC_SUBST([CPPFLAGS_FOR_BUILD])dnl
+AC_SUBST([LDFLAGS_FOR_BUILD])dnl
+])

--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -301,7 +301,6 @@ dat/scripts/prng.lua
 dat/scripts/proximity.lua
 dat/scripts/selectiveclear.lua
 dat/snd/music.lua
-po/credits.pot
 src/ai.c
 src/ai.h
 src/array.c
@@ -621,4 +620,3 @@ src/utf8.c
 src/utf8.h
 src/weapon.c
 src/weapon.h
-po/xml.pot

--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1,0 +1,2 @@
+utils/gimp/mkplanet/mkplanet.scm
+utils/gimp/mksprite/mksprite.scm

--- a/po/POTFILES_COMBINED.in
+++ b/po/POTFILES_COMBINED.in
@@ -301,7 +301,6 @@ dat/scripts/prng.lua
 dat/scripts/proximity.lua
 dat/scripts/selectiveclear.lua
 dat/snd/music.lua
-po/credits.pot
 src/ai.c
 src/ai.h
 src/array.c
@@ -2714,3 +2713,4 @@ dat/ssys/zintar.xml
 dat/ssys/zylex.xml
 dat/start.xml
 dat/tech.xml
+po/credits.pot

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -11,12 +11,11 @@ naev_DEPENDENCIES = $(NAEV_DEPENDENCIES)
 
 if HAVE_SHADERS_C_GEN
 # Compile shaders_c_gen for the build environment.
-# XXX - Should use $(BUILD_EXEEXT) instead of "$(EXEEXT)", but (cf. configure.ac) we're not using the nice "CC_FOR_BUILD" macro.
-shaders.gen.c shaders.gen.h: shaders_c_gen$(EXEEXT)
+shaders.gen.c shaders.gen.h: shaders_c_gen$(BUILD_EXEEXT)
 	./$<
-shaders_c_gen$(EXEEXT): shaders_c_gen.c
+shaders_c_gen$(BUILD_EXEEXT): shaders_c_gen.c
 	$(CC_FOR_BUILD) $(CFLAGS_FOR_BUILD) $< -o $@
-CLEANFILES += shaders.gen.c shaders.gen.h shaders_c_gen$(EXEEXT)
+CLEANFILES += shaders.gen.c shaders.gen.h shaders_c_gen$(BUILD_EXEEXT)
 BUILT_SOURCES = shaders.gen.c shaders.gen.h
 endif
 

--- a/src/land_outfits.c
+++ b/src/land_outfits.c
@@ -197,7 +197,7 @@ void outfits_open( unsigned int wid, Outfit **outfits, int noutfits )
          w - (20 + iw + 20) - 200 - 20, 320, 0, "txtDescShort", &gl_smallFont, NULL, NULL );
 
    window_addText( wid, 20 + iw + 20, -60-128-10-32,
-         60, 160, 0, "txtSDesc", &gl_smallFont, NULL,
+         90, 160, 0, "txtSDesc", &gl_smallFont, NULL,
          _("\anOwned:\a0\n"
          "\n"
          "\anSlot:\a0\n"
@@ -207,8 +207,8 @@ void outfits_open( unsigned int wid, Outfit **outfits, int noutfits )
          "\anPrice:\a0\n"
          "\anMoney:\a0\n"
          "\anLicense:\a0\n") );
-   window_addText( wid, 20 + iw + 20 + 60, -60-128-10-32,
-         w - (20 + iw + 20 + 60), 160, 0, "txtDDesc", &gl_smallFont, NULL, NULL );
+   window_addText( wid, 20 + iw + 20 + 90, -60-128-10-32,
+         w - (20 + iw + 20 + 90), 160, 0, "txtDDesc", &gl_smallFont, NULL, NULL );
    window_addText( wid, 20 + iw + 20, -60-128-10-160-32,
          w-(iw+80), MAX(180, h - 500), /* TODO: Size exactly and resize instead of moving? */
 	 0, "txtDescription", &gl_smallFont, NULL, NULL );
@@ -394,7 +394,7 @@ void outfits_update( unsigned int wid, char* str )
       /* Reposition. */
       th = 128;
       window_moveWidget( wid, "txtSDesc", 20+iw+20, -60-th-20-32 );
-      window_moveWidget( wid, "txtDDesc", 20+iw+20+60, -60-th-20-32 );
+      window_moveWidget( wid, "txtDDesc", 20+iw+20+90, -60-th-20-32 );
       window_moveWidget( wid, "txtDescription", 20+iw+20, -240-32);
       return;
    }
@@ -456,7 +456,7 @@ void outfits_update( unsigned int wid, char* str )
    window_modifyText( wid, "txtDescShort", _(outfit->desc_short) );
    th = MAX( 128, gl_printHeightRaw( &gl_smallFont, 280, _(outfit->desc_short) ) );
    window_moveWidget( wid, "txtSDesc", 20+iw+20, -60-th-20-32 );
-   window_moveWidget( wid, "txtDDesc", 20+iw+20+60, -60-th-20-32 );
+   window_moveWidget( wid, "txtDDesc", 20+iw+20+90, -60-th-20-32 );
    th += gl_printHeightRaw( &gl_smallFont, 250, buf );
    window_moveWidget( wid, "txtDescription", 20+iw+20, -60-th-20-32 );
 }

--- a/src/map_find.c
+++ b/src/map_find.c
@@ -669,7 +669,7 @@ static void map_addOutfitDetailFields(unsigned int wid, int x, int y, int w, int
    window_addText( wid, iw + 128 + 20, -60 - gl_defFont.h - 20,
          280, 160, 0, "txtDescShort", &gl_smallFont, NULL, NULL );
    window_addText( wid, iw+20, -60-128-10,
-         60, 160, 0, "txtSDesc", &gl_smallFont, NULL,
+         90, 160, 0, "txtSDesc", &gl_smallFont, NULL,
          _("\anOwned:\a0\n"
          "\n"
          "\anSlot:\a0\n"
@@ -680,7 +680,7 @@ static void map_addOutfitDetailFields(unsigned int wid, int x, int y, int w, int
          "\anMoney:\a0\n"
          "\anLicense:\a0\n") );
    window_addText( wid, iw+20, -60-128-10,
-         280, 160, 0, "txtDDesc", &gl_smallFont, NULL, NULL );
+         w - (20 + iw + 20 + 90), 160, 0, "txtDDesc", &gl_smallFont, NULL, NULL );
    window_addText( wid, iw+20, -60-128-10-160,
          w-(iw+80), 180, 0, "txtDescription",
          &gl_smallFont, NULL, NULL );
@@ -739,13 +739,13 @@ static void map_showOutfitDetail(unsigned int wid, char* wgtname, int x, int y, 
          mass,
          buf2,
          buf3,
-         (outfit->license != NULL) ? outfit->license : _("None") );
+         (outfit->license != NULL) ? _(outfit->license) : _("None") );
    window_modifyText( wid, "txtDDesc", buf );
    window_modifyText( wid, "txtOutfitName", _(outfit->name) );
    window_modifyText( wid, "txtDescShort", _(outfit->desc_short) );
    th = MAX( 128, gl_printHeightRaw( &gl_smallFont, 280, _(outfit->desc_short) ) );
    window_moveWidget( wid, "txtSDesc", iw+20, -60-th-20 );
-   window_moveWidget( wid, "txtDDesc", iw+20+60, -60-th-20 );
+   window_moveWidget( wid, "txtDDesc", iw+20+90, -60-th-20 );
    th += gl_printHeightRaw( &gl_smallFont, 280, buf );
    window_moveWidget( wid, "txtDescription", iw+20, -60-th-40 );
 }

--- a/src/nlua_gui.c
+++ b/src/nlua_gui.c
@@ -53,7 +53,7 @@ static const luaL_Reg guiL_methods[] = {
    { "mouseMoveEnable", guiL_mouseMoveEnable },
    { "menuInfo", guiL_menuInfo },
    { "menuSmall", guiL_menuSmall },
-   { "mapOverlayBounds", guiL_setMapOverlayBounds },
+   { "setMapOverlayBounds", guiL_setMapOverlayBounds },
    {0,0}
 }; /**< GUI methods. */
 
@@ -433,7 +433,7 @@ static int guiL_menuSmall( lua_State *L )
  *    @luatparam number right Right boundary in pixels
  *    @luatparam number bottom Bottom boundary in pixels
  *    @luatparam number left Left boundary in pixels
- * @luafunc mapOverlayBounds( top, right, bottom, left )
+ * @luafunc setMapOverlayBounds( top, right, bottom, left )
  */
 static int guiL_setMapOverlayBounds( lua_State *L )
 {

--- a/utils/update-po.sh
+++ b/utils/update-po.sh
@@ -13,7 +13,6 @@ set -x
 
 # General file
 TMPFILE=$(mktemp)
-echo "po/credits.pot" >> "$TMPFILE"
 find src/ -name "*.[ch]" -not \( -name "glue_macos.h" -or -name "khrplatform.h" -or -name "shaders*gen*" \) >> "$TMPFILE"
 find dat/ -name "*.lua" >> "$TMPFILE"
 
@@ -29,9 +28,9 @@ find dat/ships -name "*.xml" >> "$TMPFILE"
 find dat/ssys -name "*.xml" >> "$TMPFILE"
 
 cat "$TMPFILE" | LC_ALL=C sort | tee -a "$ROOT/po/POTFILES_COMBINED.in" > "$ROOT/po/POTFILES_XML.in"
-echo "po/xml.pot" >> "$ROOT/po/POTFILES.in"
 
 rm "$TMPFILE" # clean-up
 
 # Pull strings from the "intro" and "AUTHORS" files (inputs to credit rolls) into "credits.pot".
 awk -f "$ROOT/utils/update-po-credits.awk" dat/intro dat/AUTHORS > "$ROOT/po/credits.pot"
+echo "po/credits.pot" >> "$ROOT/po/POTFILES_COMBINED.in"

